source/hardware/controllers/pioneer_ddj_sb3.rst

```diff
@@ -0,0 +1,153 @@
+12   :hwlabel:`SHIFT` + :hwlabel:`PLAY/PAUSE` button    Return to temporary cue point
+13   :hwlabel:`CUE` button                              Set, play and call out cue points
+13   :hwlabel:`SHIFT` + :hwlabel:`CUE` button           Return to cue point and stop or load previous track
+14   :hwlabel:`SYNC` button                             Match tempo and phase of adjacent deck
+14   :hwlabel:`SHIFT` + :hwlabel:`SYNC` button          Turn off sync mode
+15   :hwlabel:`SHIFT` + button                          Switch function of controls
+16   :hwlabel:`DECK X`                                  Switch deck
+17   :hwlabel:`KEY LOCK`                                Toggle key lock
+17   :hwlabel:`SHIFT` + :hwlabel:`KEY LOCK`             Toggle tempo slider range
+===  =================================================  ============================================================================================
+
+
+.. _pioneer-ddj-sb3-mixer:
+
+Mixer section (p. 8)
+~~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_sb3_mixer.svg
+   :align: center
+   :width: 50%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (mixer section)
+   :figclass: pretty-figures
+
+   Mixer section (p. 8)
+
+===  =====================================================  ============================================================================================
+No.  Control                                                Function
+===  =====================================================  ============================================================================================
+1    :hwlabel:`TRIM` knob                                   Adjust inidvidual channel output gain
+2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs      Adjust high/mid/low-frequencies
+3    :hwlabel:`FILTER` knob                                 Low pass / high pass filter for each channel
+4    :hwlabel:`HEADPHONE CUE 1` button                      Toggle headphone pre-fader listening of left deck.
+4    :hwlabel:`HEADPHONE CUE 2` button                      Toggle headphone pre-fader listening of right deck.
+5    Channel faders                                         Adjust the output level for each channel
+6    Cross fader                                            Fade between left and right deck
+7    :hwlabel:`MASTER` knob                                 Adjust master output level
+8    :hwlabel:`HEADPHONES` knob                             Adjust headphones output level
+9    :hwlabel:`MASTER` button                               Output master to headphones
+===  =====================================================  ============================================================================================
+
+Performance Pads
+~~~~~~~~~~~~~~~~
+
+You can use the Pad Mode Select buttons to select a mode for the
+performance pads. Performance pads are numbered from top left to bottom right. First row is 1-4 and the second row is 5-8.
```

Comment:
Is there only a single pad mode available? If so, this paragraph is misleading because you can't actually select pad modes other than "Hot Cue Mode". If you can select other modes, please document them, too.

Reply:
You're right, there are other modes. I will add those as well.

Reply:
I have added additional documentation for the missing modes.